src/java/fr/paris/lutece/plugins/workflow/modules/formspdf/business/FormsPDFTaskTemplateDAO.java

```diff
@@ -0,0 +1,113 @@
+package fr.paris.lutece.plugins.workflow.modules.formspdf.business;
+
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+import fr.paris.lutece.plugins.workflow.modules.formspdf.service.FormsPDFPlugin;
+import fr.paris.lutece.util.sql.DAOUtil;
+
+public class FormsPDFTaskTemplateDAO implements IFormsPDFTaskTemplateDAO {
+	
+	 private static final String SQL_QUERY_SELECTALL = "SELECT id_template, name, content FROM workflow_task_formspdf_template";
+
+	 private static final String SQL_QUERY_SELECT = SQL_QUERY_SELECTALL + " WHERE id_template = ?";
+	 
+	 private static final String SQL_QUERY_INSERT = "INSERT INTO workflow_task_formspdf_template ( name, content ) VALUES ( ?, ? ) ";
+	 
+	 private static final String SQL_QUERY_DELETE = "DELETE FROM workflow_task_formspdf_template WHERE id_template = ? ";
+	 
+	 private static final String SQL_QUERY_UPDATE = "UPDATE workflow_task_formspdf_template SET name = ?, content = ? WHERE id_template = ?";
+	 
+	 @Override
+	 public void insert(FormsPDFTaskTemplate formsPDFTaskTemplate)
+	 {
+		try ( DAOUtil daoUtil = new DAOUtil( SQL_QUERY_INSERT, Statement.RETURN_GENERATED_KEYS, FormsPDFPlugin.getPlugin( ) ) )
+        {
+			int nIndex = 0;
+			daoUtil.setString( ++nIndex, formsPDFTaskTemplate.getName() );
+			daoUtil.setString( ++nIndex, formsPDFTaskTemplate.getContent() );
+			
+			daoUtil.executeUpdate( );
+			
+			if ( daoUtil.nextGeneratedKey( ) )
+			{
+				formsPDFTaskTemplate.setId( daoUtil.getGeneratedKeyInt( 1 ) );
+			}
+			daoUtil.free( );
+        }
+	 }
+	 
+	 @Override
+	 public void store(FormsPDFTaskTemplate formsPDFTaskTemplate)
+	 {
+		@SuppressWarnings( "resource" )
+		DAOUtil daoUtil = new DAOUtil( SQL_QUERY_UPDATE, FormsPDFPlugin.getPlugin( ) );
+		
+		int nIndex = 0;
+		daoUtil.setString( ++nIndex, formsPDFTaskTemplate.getName( ) );
+		daoUtil.setString( ++nIndex, formsPDFTaskTemplate.getContent( ) );
+		
+		daoUtil.setInt( ++nIndex, formsPDFTaskTemplate.getId( ) );
+		
+		daoUtil.executeUpdate( );
+		daoUtil.free( );
+	 }
+	 
+	 @Override
+	 public FormsPDFTaskTemplate load( int nIdTemplate )
+	 {
+		@SuppressWarnings( "resource" )
```

Comment:
use try with resource

Reply:
Thank you for your valuable feedback ! 

We apologize for the delay in our response. I am pleased to inform you that all issues have been resolved.
Regarding the late response, I would like clarify that I have managed several projects outside the Forms scope see https://github.com/lutece-platform/lutece-wf-module-workflow-formstopdf/pull/1#issuecomment-1629002646  However, my colleague Louis, who was working with us on this project, is no longer with us. Despite this, I have taken responsibility for resolving all the points you have raised.

Please do not hesitate to contact me or our dedicated Forms team if you have any further questions or concerns.